server/src/main/java/org/elasticsearch/search/SearchService.java

```diff
@@ -765,7 +762,7 @@ private void parseSource(DefaultSearchContext context, SearchSourceBuilder sourc
             } catch (IOException e) {
                 throw new SearchContextException(context, "failed to create sort elements", e);
             }
-        }
+        }NestedQueryBuilderTestsNestedQueryBuilderTests
```

Comment:
leftover I guess :) I do this all the time

Reply:
lol, yep just before I commit